### DISCUSSION TITLE
Multiple tabs with different ssh servers in one window

### DISCUSSION THREAD

**Discussion by vecinohk** [Eugeny/tabby#8030] [Q&A] [answered]
Hi guys,

can someone help me please? Is it possible to open multiple tabs with different ssh servers in one window? I know that I can open multiple tabs in one window, but only with the same guest. Or did I miss something?

Sometimes I work with one ssh access and I need to open another ssh access to copy the configuration, for example. Now I have to start a new instance of Tabby with other ssh server and place split windows next to each other. 

Screenshot to better explain how I'm using it now.: [https://ibb.co/c8g1PgP](https://ibb.co/c8g1PgP)

Thanks!

**Comment by Eugeny** [comment]
* You can drag another tab into your current tab and attach it to the side, or:
* You can split the current tab, and then use the "switch profile" command (in the context menu or the command palette) to replace it with a different profile

**Comment by vecinohk** [comment]
Thank you for answer. I tried the dragging as the first logical thing that came to mind, but it didn't work. I figured out that in the settings - Windows in the item I must not have "Window frame" set to Full then it doesn't work. Now that I've changed it to Thin, I can drag and drop, but I can only have the window at the top or bottom. I can't drag it to the right, for example. Is that right?

Edit: When I try the interactive demo at https://tabby.sh/, I can drag tabs to all sides there - it works as it should. So it's not working properly for me. I have something wrong.

I have Windows 11 [Version 10.0.22621.1265]
Tabby: Alpha 189

Thanks!

**Comment by vecinohk** [comment]
Today I tried it on Arch linux and it works completely fine there. So the problem only occurs on Windows 11.

**Comment by vecinohk** [comment, marked as answer]
The **Focus follows mouse** option must be **disabled**. Then everything is fine.
